src/components/Login/LoginContainer.tsx

```diff
@@ -274,85 +275,96 @@ export const InitialisationContainer: FunctionComponent<
     }
   };
 
+  const [shouldBlock, setShouldBlock] = useState(true);
+  useEffect(() => {
+    setShouldBlock(true);
+  }, []);
+
```

Comment:
blocks when user navigates to login (AKA add new campaign)

https://github.com/rationally-app/mobile-application/assets/43963814/534e7b57-9e95-4b26-9fb0-be2069be792d